src/Psl/Xml/constants.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Psl\Xml;
+
+use const LIBXML_ERR_ERROR;
+use const LIBXML_ERR_FATAL;
+use const LIBXML_ERR_NONE;
+use const LIBXML_ERR_WARNING;
+
+const ERR_ERROR = LIBXML_ERR_ERROR;
+const ERR_FATAL = LIBXML_ERR_FATAL;
+const ERR_NONE = LIBXML_ERR_NONE;
+const ERR_WARNING = LIBXML_ERR_WARNING;
```

Comment:
I don't think there's a point of declaring these constants.

should we move them to `Xml\Error` class instead? 

however, having `Error::LEVEL_ERROR` doesn't make sense, if it's an error, why would it have a warning level instead of a error level? shouldn't that be in a different subclass of a parent class that repesent all of `Warning`, `Error`, and `FatalError`?

i think another idea would be to rename `Error`/`ErrorCollection` to `Issue`/`IssueCollection`, as `Issue::warning`/`Issue::error`/`issue::fatal` make more sense.

WDYT?

Reply:
That's possible ... and results in a cleaner API. will do!

Reply:
P.S: `none` shouldn't be included, if it's 'none', it's shouldn't be part of the `IssueCollection` :)